AS_920_923_TTN_VN.yml

```diff
@@ -0,0 +1,93 @@
+band-id: AS_923
+sub-bands:
+- min-frequency: 920000000
+  max-frequency: 923000000
+duty-cycle: 0.01             # Duty cycle for this sub-band (optional; default: 1)
+max-eirp: 16
+uplink-channels:
+- frequency: 921400000
+  min-data-rate: 0
+  max-data-rate: 5
+  radio: 0
+- frequency: 921600000
+  min-data-rate: 0
+  max-data-rate: 5
+  radio: 0
+- frequency: 921800000
+  min-data-rate: 0
+  max-data-rate: 5
+  radio: 0
+- frequency: 922000000
+  min-data-rate: 0
+  max-data-rate: 5
+  radio: 0
+- frequency: 922200000
+  min-data-rate: 0
+  max-data-rate: 5
+  radio: 0
+- frequency: 922400000
+  min-data-rate: 0
+  max-data-rate: 5
+  radio: 1
+- frequency: 922600000
+  min-data-rate: 0
+  max-data-rate: 5
+  radio: 1
+- frequency: 922800000
+  min-data-rate: 0
+  max-data-rate: 5
+  radio: 1
+downlink-channels:
+- frequency: 921400000
+  min-data-rate: 0
+  max-data-rate: 5
+- frequency: 921600000
+  min-data-rate: 0
+  max-data-rate: 5
+- frequency: 921800000
+  min-data-rate: 0
+  max-data-rate: 5
+- frequency: 922000000
+  min-data-rate: 0
+  max-data-rate: 5
+- frequency: 922200000
+  min-data-rate: 0
+  max-data-rate: 5
+- frequency: 922400000
+  min-data-rate: 0
+  max-data-rate: 5
+- frequency: 922600000
+  min-data-rate: 0
+  max-data-rate: 5
+- frequency: 922800000
+  min-data-rate: 0
+  max-data-rate: 5
+
+lora-standard-channel:
+  frequency: 923000000
+  data-rate: 6
+  radio: 1
+#fsk-channel:
+#  frequency: 921200000
+#  data-rate: 7
+#  radio: 0
+radios:
+- enable: true
+  chip-type: SX1257
+  frequency: 921800000
+  rssi-offset: -166
+  tx:
+    min-frequency: 920000000
+    max-frequency: 923000000
+- enable: true
+  chip-type: SX1257
+  frequency: 922800000
+  rssi-offset: -166
+clock-source: 1
+rx2-channel:                   # Rx2 channel (optional)
+  frequency: 921400000
+  min-data-rate: 0
+  max-data-rate: 5
+  radio: 0
+rx2-default-data-rate: 2       # Default data rate index of Rx2 (optional)
+max-eirp: 16
```

Comment:
`max-eirp` is in here twice

Reply:
You are right, first we plan 20dbm for rx2, and 16dbm for sub-band.

Reply:
Also, you enter there 16 dBm maximum EIRP, not 20 dBm.